dependencies.props

```diff
@@ -157,7 +157,7 @@
     </StaticDependency>
 
     <StaticDependency Include="Microsoft.TargetingPack.Private.WinRT">
-      <Version>1.0.3</Version>
```

Comment:
A way to check is to give Version a garbage value (say, `9.9.9`) and run `msbuild /t:VerifyDependencies`. If there are no verification errors, then this item isn't doing anything useful.